portlets/src/main/frontend/src/apps/profileStats/components/SpaceDrawer.vue

```diff
@@ -393,6 +395,11 @@ export default {
         this.spacesSuggestionsList = data.items;
       });
     },
+    removeLeavedSpace(spaceName) {
+      const filtered = this.spaces.filter(space => space.prettyName !== spaceName);
```

Comment:
Could you use space.id or space.groupId instead. They are not changeable and better for testing

Reply:
The only way is to use the space.prettyName because at UISpaceInfoPopup.js we only have acces to the space's prettyName since the rest service takes as parameters a set of space.prettyName:USerName:role 
https://github.com/Meeds-io/social/blob/a658db19e77272055901137153a2aac96a27c987/component/service/src/main/java/org/exoplatform/social/rest/impl/spacemembership/SpaceMembershipRestResourcesV1.java#L343

Reply:
ok thanks